specification/cognitiveservices/data-plane/HealthInsights/readme.md

```diff
@@ -0,0 +1,25 @@
+# Cognitive Services Health Insights SDK
```

Comment:
This should be removed @lmazuel

Reply:
We might actually still need to keep the readme.md as well for some other processing.

Reply:
Got it